es/separable_natural_es.py

```diff
@@ -37,6 +50,11 @@ def optimize(func, mu, sigma,
     history_pop = []
     history_fitness = []
 
+    n_total_individuals = population_size * (1 + int(mirrored_sampling))
+    logger.info('starting evolution with {} individuals per generation on {} threads'.format(n_total_individuals, parallel_threads))
+    if (parallel_threads is None and n_total_individuals > 1) or (parallel_threads is not None and n_total_individuals > parallel_threads):
+        logger.warning('more individuals than parallel threads. expect long runtime')
```

Comment:
"expect long runtime" sounds a bit casual. What about 

    "more individuals ({}) than parallel threads ({}). consider increasing the number of parallel threads.".format(n_total_individuals, parallel_threads)

Reply:
Is it a convention to use only lowercase in logging messages?